server/core/src/main/java/dev/slimevr/tracking/processor/Bone.kt

```diff
@@ -58,6 +58,14 @@ class Bone(val boneType: BoneType) {
 		headNode.update()
 	}
 
+	/**
+	 * Computes the rotations and positions of this bone.
+	 */
+	fun updateThisNode() {
+		headNode.updateThisNode()
+		tailNode.updateThisNode()
+	}
```

Comment:
This is unsafe in regards to the bone hierarchy. Please see my earlier comment in **HumanSkeleton** regarding a refactor of how constraints are applied.